src/store/Developers.js

```diff
@@ -49,7 +50,23 @@ const actions = {
   }
 };
 
-const getters = {};
+const getters = {
+  newDevelopers(state) {
+    const newDevs = clone(state.developers).filter(dev => {
```

Comment:
Why clone ?

Reply:
I move it to the map, but we need to clone it because we need to mutate directly the `position` key of the state.